erpnext/regional/italy/utils.py

```diff
@@ -191,6 +191,13 @@ def sales_invoice_validate(doc):
 	else:
 		validate_address(doc.company_address, "Company")
 
+	company_fiscal_regime = frappe.get_cached_value("Company", doc.company, 'fiscal_regime')
+	if not company_fiscal_regime:
+		frappe.throw(_("Fiscal Regime is mandatory, kindly set the fiscal regime in the company {0}")
+			.format(doc.company))
+	else:
+		doc.company_fiscal_regime = company_fiscal_regime
```

Comment:
what if fiscal regime is set manually in `doc`? overwrite?

Reply:
It's read only field in the sales invoice